massa-bootstrap/src/establisher.rs

```diff
@@ -29,15 +33,37 @@ pub mod types {
     /// connection establisher
     pub type Establisher = DefaultEstablisher;
 
+    fn normalize_ip(ip: IpAddr) -> IpAddr {
```

Comment:
Let's add a little bit of documentation to explain why we dont just do to_canonical